public/app/features/alerting/unified/components/receivers/TemplateForm.tsx

```diff
@@ -37,6 +39,7 @@ interface Props {
   alertManagerSourceName: string;
   provenance?: string;
 }
+export const isDuplicating = (location_: Location) => location_.pathname.endsWith('/duplicate');
```

Comment:
any particular reason why we're prefixing here with `_`?

Reply:
you're right, we can remove the prefix (at the beginning I had the method inside the component where we had another location const defined, that's why I added the prefix. But not it's not necessary)